README.asc

```diff
@@ -7,7 +7,7 @@ You can find this book online at: https://git-scm.com/book
 Like the first edition, the second edition of Pro Git is open source under a Creative Commons license.
 
 A couple of things have changed since open sourcing the first edition.
-For one, we've moved from Markdown to the amazing Asciidoc format for the text of the book.
+For one, we've moved from Markdown to the amazing AsciiDoc format for the text of the book; here's an https://powerman.name/doc/asciidoc[AsciiDoc quick reference].
```

Comment:
Let's use this one instead: https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/

Reply:
Sounds good!